test/model/content_test.dart

```diff
@@ -139,6 +139,48 @@ class ContentExample {
     '<p><span class="silent user-group-mention" data-user-group-id="186">test-empty</span></p>',
     const UserMentionNode(nodes: [TextNode('test-empty')]));
 
+  static final channelWildcardMentionPlain = ContentExample.inline(
+    'plain channel wildcard @-mention',
+    "@**all**",
+    expectedText: '@all',
+    '<p><span class="user-mention channel-wildcard-mention" data-user-id="*">@all</span></p>',
+    const UserMentionNode(nodes: [TextNode('@all')]));
+
+  static final channelWildcardMentionSilent = ContentExample.inline(
+    'silent channel wildcard @-mention',
+    "@_**everyone**",
+    expectedText: 'everyone',
+    '<p><span class="user-mention channel-wildcard-mention silent" data-user-id="*">everyone</span></p>',
+    const UserMentionNode(nodes: [TextNode('everyone')]));
+
+  static final channelWildcardMentionSilentClassOrderReversed = ContentExample.inline(
+    'silent channel wildcard @-mention, class order reversed',
+    "@_**channel**", // (hypothetical server variation)
+    expectedText: 'channel',
+    '<p><span class="silent user-mention channel-wildcard-mention" data-user-id="*">channel</span></p>',
+    const UserMentionNode(nodes: [TextNode('channel')]));
+
+  static final legacyChannelWildcardMentionPlain = ContentExample.inline(
```

Comment:
Yeah, this naming is good: the current/new way gets the simple name, and the old way gets the longer "legacy" name.